components/utils.py

```diff
@@ -334,6 +345,10 @@ def start(service_name):
     def stop(service_name):
         sudo(['systemctl', 'stop', service_name])
 
+    @staticmethod
+    def reload_configuration():
```

Comment:
Will do- I named it like this because it actually makes more sense to anyone who doesn't work with systemd much. I'll add a comment to make it clear for anyone in that camp.